components/Modal/DeleteBlueprint.js

```diff
@@ -49,11 +51,13 @@ class DeleteBlueprint extends React.Component {
               <button
                 type="button"
                 className="btn btn-default"
+                id="btn-default"
```

Comment:
`id` shouldn't be added here

In this case it's fairly safe to assume that this is the only element that would have the class `btn-default`, so for the test, we can use the class selector: `.btn-default`